### ISSUE TITLE
admin datatbles not showing HTML properly

### ISSUE THREAD

**Issue by alhad15** [sgsoft-studio/webed#40]
Hello 
i have installed CMS XAMP , after installation all pages where datatable is shown like Theme Management are showing encoded HTML 
![image](https://user-images.githubusercontent.com/1075139/28744112-dc176b44-7477-11e7-83b6-02a174245815.png)

**Comment by duysolo** [comment]
Did you `composer update`?

**Comment by alhad15** [comment]
yes i did it, but no change

**Comment by duysolo** [comment]
Which WebEd version are you using?

**Comment by alhad15** [comment]
its 3.1.56
i am using php 7.0.4

**Comment by duysolo** [comment]
Please check the file in
`vendor/sgsoft-studio/themes-management/src/Http/DataTables/ThemesListDataTable.php` at line 72.

The column `thumbnail` should be set as raw column.
```
->rawColumns(['description', 'actions', 'thumbnail'])
```

Please run these commands:
```
composer clear-cache
composer update
```
Hope it works.

**Comment by alhad15** [comment]
issue was due to bug in yajra/laravel-datatables-oracle v7.9.6  it is fixed in yajra/laravel-datatables-oracle v7.9.7
Thanks

**Comment by duysolo** [comment]
Ok thanks.
